tests/fe/fe_values_view_30.cc

```diff
@@ -109,7 +109,7 @@ void test (const Triangulation<dim> &tr,
 
   MappingQGeneric<dim> mapping(1);
 
-  const QGauss<dim> quadrature(2);
+  const QGauss<dim> quadrature(fe.degree+2);
```

Comment:
The changed test passes for the old implementation as well as for the new one. So yes, changing the test changes the output.

The old implementation uses the given `Quadrature` for assembling both the `system_matrix` and the `rhs`
while the new implementation always uses `QGauss(fe_degree+2)` for the `system_matrix` and the given `Quadrature` for the `rhs`.